lib/resty/etcd/v3.lua

```diff
@@ -67,30 +94,68 @@ local function _request_uri(self, endpoint, method, uri, opts, timeout, ignore_a
         http_cli:set_timeout(timeout * 1000)
     end
 
-    local res
-    res, err = http_cli:request_uri(uri, {
-        method = method,
-        body = body,
-        headers = headers,
-        keepalive = keepalive,
-        ssl_verify = self.ssl_verify,
-        ssl_cert_path = self.ssl_cert_path,
-        ssl_key_path = self.ssl_key_path,
-    })
+    local function http_request()
```

Comment:
It seems we can move this function instead of creating it dynamically

Reply:
Thanks! Fixed